login-fire-form-behavior.html

```diff
@@ -26,6 +26,14 @@
         showResetPW: {
             type: Boolean,
             value: false
+        },
+        noToggleablePassword: {
+            type: Boolean,
+            value: false
+        },
+        reenterPassword: {
```

Comment:
This works on #72 you can see the idea here http://output.jsbin.com/cawija 

What we're planning to implement is a little different in that it isn't a toggle, it shows the password while you hold the view icon.  You can read more about the idea at http://uxmovement.com/forms/why-the-confirm-password-field-must-die/

This will be the new default but you can add re-enter password and remove the toggle if you want.